app/src/main/java/net/skyscanner/backpack/demo/compose/BadgeStory.kt

```diff
@@ -69,21 +68,25 @@ private fun BadgeRow(
             .padding(horizontal = BpkSpacing.Base, vertical = BpkSpacing.Md),
     ) {
 
-        BpkBadge(
-            text = type.toString(),
-            type = type,
-            modifier = Modifier.weight(1f).wrapContentWidth(align = Alignment.CenterHorizontally),
-        )
+        if (type !in listOf(BpkBadgeType.Success, BpkBadgeType.Warning, BpkBadgeType.Destructive)) {
+            BpkBadge(
+                text = type.toString(),
+                type = type,
+                modifier = Modifier
+                    .weight(1f)
+                    .wrapContentWidth(align = Alignment.CenterHorizontally),
+            )
+        } else {
+            Spacer(modifier = Modifier.weight(1F))
+        }
 
         BpkBadge(
             text = type.toString(),
-            modifier = Modifier.weight(1f).wrapContentWidth(align = Alignment.CenterHorizontally),
+            modifier = Modifier
+                .weight(1f)
+                .wrapContentWidth(align = Alignment.CenterHorizontally),
             type = type,
-            icon = when (type) {
-                BpkBadgeType.Warning -> BpkIcon.HelpCircle
-                BpkBadgeType.Destructive -> BpkIcon.CloseCircle
-                else -> BpkIcon.TickCircle
-            },
+            icon = BpkIcon.TickCircle,
```

Comment:
To match what Figma does here I would use a null icon for warning, destructive and success, otherwise the tick - otherwise all icons will be ticks now on the sample/docs
Also can you re-generate the docs screenshots please? btw as badge is used in some other components these will have changed too (as you can see by the snapshot tests) - can you keep the changes for these docs screenshots as well?